tests/test_admin.py

```diff
@@ -0,0 +1,41 @@
+import os
+import json
+import unittest
+from selenium import webdriver
+from selenium.webdriver.chrome.options import Options
+from selenium.common.exceptions import NoSuchElementException
+from utils import TestUtilities
+
+
+class TestAdmin(TestUtilities):
+
+    def setUp(self):
```

Comment:
I think this is repeated, it can be moved to a parent class from where both the tests will get the common code! :smile:

Reply:
Should I do the same thing to websocket?